enhancements/metering/slim-install.md

```diff
@@ -0,0 +1,210 @@
+- In most environments, metering is only querying and storing a small subset of Prometheus metrics and the total size of that datastore typically doesn't warrant the usage of a distributed query engine, like Presto.
+- In the case a user wishes to install metering in a disconnected, air-gapped, or simply on an environment that does not have access to cloud storage, metering requires the usage of a StorageClass or PersistentVolume with ReadWriteMany (RMW) access modes.
+- At a minimum, metering requires the user to define a storage configuration in the `MeteringConfig` custom resource and that leaves the door open to potential misconfigurations or difficulties setting up the pre-requisites for metering usage.
+
+### Goals
+
+- Reduce the complexity of the metering stack.
+- Reduce the overall CPU and memory footprint of the metering stack.
+- New metering installations default to enabling the slim install configuration.
+- A potential migration path is outlined.
+
+### Non-Goals
+
+- Initial support for cloud object storage.
+- Maintaining support for AWS billing integration.
+- Metering is delivered as a CVO-based Operator.
+
+## Proposal
+
+Currently, and by default, metering periodically imports and stores a small subset of the available Prometheus metrics in Hive long term, using the Prometheus query_range API. Historically, Presto did not have the ability to directly query Prometheus, but it does have the ability to integrate with Hive datasets. This results in the following workaround, where tables are needed to be first created in Hive, and then newer metrics would be inserted by Presto into existing tables in Hive using the `INSERT INTO ... VALUES(...)` operation. It's also worth noting that type of operation is particularly expensive for Presto in comparison to Postgres, which is designed as a more transactional-based system (OLTP).
+
+Due to the nature of periodic importing, most of the components are sitting idle, especially in the case of Presto which is still utilizing resources despite no queries being processed or evaluated.
+
+With the introduction of Postgres, and the removal of Hive and Presto components, we can continue to utilize the existing Prometheus metrics importer implementation, adapting to the native Postgres data types, and the result is a more streamlined approach to data flow and an overall less complex stack.
+
+This implementation would allow metering to continue using the core, existing CustomResourceDefinitions (CRDs) already exposed in previous releases:
+
+- `ReportDataSource`: exposes an existing table in Postgresql that can be used for reporting needs.
+- `ReportQuery`: defines a SQL query or view that references a `ReportDataSource` table, or a table created by a `Report` object.
+- `Report`: evaluates a templated SQL query defined in an existing `ReportQuery` custom resource and stores those query results in a Hive (now Postgres) table.
+
+At a high-level, the following must be done:
```

Comment:
If removing Presto/Hive/Hadoop/HDFS are goals, should we state that here so it doesn't seem like we'll be leaving them around unused?

Reply:
the goal is slimming down the metering resource requirements.  eliminating presto/hive/hdfs is the proposed implementation.  But if it's not clear that the implementation is removing them, yes that should be made clearer.

Reply:
@bparees Originally what I proposing was that we continue carrying logic for reconciling the current stack, on top of this proposed architecture that is toggled by a field in the `MeteringConfig` CRD, and eventually phase out the current architecture a few releases down the line. Now, it might make more sense to propose a v2 version of Metering, where there's no logic in place for reconciling HDFS/Hive/Presto and their corresponding CRDs as well. 

With this, you would ideally provide a migration path to the v2 version, but keep the installations separate (the implication being a new bundle would be created and maintained) to avoid having to provide upgradability between versions. If that path sounds reasonable, I'm mainly trying to work out how to best deprecate the previous version. Does it make sense to mark this as tech preview with the assumption all of this lands in 4.7, and GA's in 4.8?

Reply:
> Does it make sense to mark this as tech preview with the assumption all of this lands in 4.7, and GA's in 4.8?

makes sense to me